modules/oe_list_pages_open_vocabularies/tests/modules/oe_list_pages_open_vocabularies_test/oe_list_pages_open_vocabularies_test.info.yml

```diff
@@ -0,0 +1,11 @@
+name: OpenEuropa List Pages Open Vocabularies Test
```

Comment:
In this new module, let's add also handlers for core taxonomy:

```
taxonomy:
  label: 'Local taxonomy'
  target_type: taxonomy_term
```

In a `oe_list_pages_open_vocabularies_test.vocabulary_reference_handlers.yml` file.